lib/rules/no-mocha-global-arrows.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+/**
+ * @fileoverview Disallow arrow functions as arguments to Mocha globals
+ * @author Paul Melnikow
+ */
+
+var R = require('ramda'),
+    mochaFunctionNames = [
+        'describe',
+        'describe.skip',
+        'xdescribe',
+        'suite',
+        'context',
+        'xcontext',
+        'specify',
+        'xspecify',
+        'it',
+        'it.only',
+        'xit',
+        'test',
+        'test.only',
+        'specify',
+        'specify.only',
+        'before',
+        'after',
+        'beforeEach',
+        'afterEach'
+    ];
+
+module.exports = function (context) {
+    function isLikelyMochaGlobal(scope, name) {
+        return !R.find(R.propEq('name', name), scope.variables);
+    }
+
+    return {
+        CallExpression: function (node) {
+            var callee = node.callee,
+                fnArg;
+
+            if (callee && callee.name && mochaFunctionNames.indexOf(callee.name) > -1) {
```

Comment:
This doesn't work for function names like `it.only`. See how this is done in [other rules](https://github.com/lo1tuma/eslint-plugin-mocha/blob/2f560171dd691ae444eb0a7ea819374a9497df50/lib/rules/handle-done-callback.js#L19-L25).